src/Tasks/CheckJobHealthTask.php

```diff
@@ -38,11 +38,21 @@ public function getDescription()
     public function run($request)
     {
         $queue = $request->requestVar('queue') ?: QueuedJob::QUEUED;
+        $jobHealth = $this->getService()->checkJobHealth($queue);
 
-        $stalledJobCount = $this->getService()->checkJobHealth($queue);
+        $unhealthyJobCount = 0;
 
-        echo $stalledJobCount === 0 ? 'All jobs are healthy' : 'Detected and attempted restart on ' . $stalledJobCount .
-            ' stalled jobs';
+        foreach ( $jobHealth as $type => $IDs ) {
+            $count = count($IDs);
+            echo 'Detected and attempted restart on ' . $count . ' ' . $type . ' jobs';
+            $unhealthyJobCount = $unhealthyJobCount + $count;
+        }
+
+        if( $unhealthyJobCount === 0 ) {
+            echo 'All jobs are healthy';
+        } else {
+            die(1);
```

Comment:
Are you sure we'd want to `die(1)` here? Previous implementation did not die

Reply:
This sets the exit code to be '1'. This would be useful for CLI implementations so you can tell if `All jobs are healthy` from the exit code.

Reply:
OK makes sense. What happens if you run this from a browser?

Reply:
PHP will halt execution, but as it's the end of the Task then it wont appear any different to the browser.

Reply:
https://www.php.net/manual/en/function.exit.phpdie & exit are also equivalent;

Reply:
At the moment there is no die, so this is new behaviour isn't it? `die` will also halt all post-request execution, which might be required. I don't think we should start adding `exit`/`die` calls if they aren't there already...

We could throw an exception which I believe will give an error code on CLI and would allow the post-execution scripts to run.

Reply:
True, I've updated it. Although throwing an exception will also halt all post-request execution?

Reply:
It won't because all the catch / finally blocks will still run